src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql

```diff
@@ -0,0 +1,665 @@
+		from (
+			   	select 		substr(h1.pool_number,1,3) as "loc_code",  -- JDB-5346 see comments above
+							jp.juror_number,
+							case
+									when j.juror_number is null then 'Paper'
+									when r.reply_type = 'Digital' then 'Online'
+									else 'Paper'
+							end as "method",
+							r.date_received as "response_date", -- digital plus paper responses but the latter is only those receieved post Juror Modernisation go_live
+							min(h1.date_created) as "summons_date",
+							min(h2.date_created) as "processed_date"
+				from 		juror_mod.juror j
+				join 		juror_mod.juror_pool jp
+					on 		jp.juror_number = j.juror_number
+				join 		juror_mod.juror_history h1
+					on 		h1.juror_number = j.juror_number
+							and h1.history_code = 'RSUM'
+				left join	juror_mod.juror_history h2
+					on 		h2.juror_number = jp.juror_number
+							and h2.history_code <> 'RSUM' -- ignore summons
+							and h2.history_code <> 'RNRE' -- ignore reminder letters
+							and h2.history_code <> 'PUND' -- Fix for JDB-4621: Undeliverable event is not a response to the summons
+							and h2.history_code <> 'PREA' -- JDB-5349 : ignore the pool reasignment
+							and h2.history_code <> 'RSUP' -- JDB-5374 : ignore summons reprinted
+				left join	juror_mod.juror_response r
+					on 		r.juror_number = j.juror_number
+				where 		jp.pool_number in (select p.pool_no from juror_mod.pool p where p.return_date >= current_date - (no_of_months || ' month')::interval)
+							and jp.is_active = true -- JDB-5346 see comments above
+							and (j.summons_file is null or j.summons_file <> 'Disq. on selection')
+							and h1.date_created > current_date - (no_of_months || ' month')::interval  -- exclude jurors summoned more than n months ago
+							and coalesce(r.date_received, h2.date_created) is null -- exclude responded jurors
+				group by 	substr(h1.pool_number,1,3),
+							jp.juror_number,
+							case
+									when j.juror_number is null then 'Paper'
+									when r.reply_type = 'Digital' then 'Online'
+									else 'Paper'
+							end,
+							r.date_received
+				order by 	jp.juror_number
+			) s
+		-- exclude responded jurors
+		where 		coalesce(response_date, processed_date) is null
+   		group by	summons_month, s.loc_code);
+
+exception
+
+	when others then
+    	get stacked diagnostics	v_text_var1 = message_text,
+                            	v_text_var2 = pg_exception_detail,
+                            	v_text_var3 = pg_exception_hint;
+
+	raise notice '%', 'not_responded failed - error:->' || v_text_var1 || '|' || v_text_var2 || '|' || v_text_var3;
+
+	rollback;
+
+end;
+
+$$;
+
+
+create or replace procedure juror_dashboard.response_times(no_of_months int)
+language plpgsql as
+
+$$
+/*
+ * Populates juror_dashboard.stats_response_times used by the Juror Digital performance dashboard
+ *
+ * Replaces the latest n months of summons months as specified by the no_of_months input parameter
+ *
+ * It is recommended to use use no_of_months = 6
```

Comment:
Could we update the comment to remove the duplicated use please?

Reply:
Removed